src/vocab-license.ttl

```diff
@@ -185,6 +185,10 @@ license:LGPL-3_0 a skos:Concept ;
   skos:inScheme <https://ontology.okp4.space/thesaurus/license> ;
   skos:prefLabel "GNU Lesser General Public License 3.0"@en .
 
+license:LO-FR-1_0 a skos:Concept ;
+  skos:inScheme <https://ontology.okp4.space/thesaurus/license> ;
+  skos:prefLabel "Open License 1.0"@en .
+
```

Comment:
Sorry @CedrickArmel , a bit late, but it's not quite complete. If we define a new vocabulary term, we have to add it to the corresponding collection, here in: `license:dataLicenses`. Can you please take care of it in another PR? Thanks to you. :pray: